app/controllers/verify/sessions_controller.rb

```diff
@@ -99,12 +99,8 @@ def process_failure
       end
     end
 
-    def view_model(error: nil)
-      Verify::SessionsNew.new(
-        error: error,
-        remaining_attempts: remaining_idv_attempts,
-        idv_form: idv_form
-      )
+    def view_model_class
+      Verify::SessionsNew
```

Comment:
whoops turns out the shared `view_model` helper method I defined was not included in this controller, fixed in 57b53cf